src/http/structs.rs

```diff
@@ -92,6 +99,64 @@ pub struct EntityDatabasePath {
     pub database: String,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct EntityQueryPath {
```

Comment:
could call it EntityPath since it could use used in other paths with only Entity slug as well

Reply:
Done :)